qiskit/optimization/algorithms/grover_optimizer.py

```diff
@@ -197,16 +196,11 @@ def solve(self, problem: QuadraticProgram) -> OptimizationResult:
                 loops_with_no_improvement += 1
                 rotation_count = int(np.ceil(aqua_globals.random.uniform(0, m - 1)))
                 rotations += rotation_count
-
                 # Apply Grover's Algorithm to find values below the threshold.
-                if rotation_count > 0:
-                    # TODO: Utilize Grover's incremental feature - requires changes to Grover.
```

Comment:
So this removed the TODO which was referenced in #1309 as something that needed to be investigated - is it still the case that using incremental mode needs looking at further. If this PR gets merged before the WIP evidently the TODO will be 'lost' so maybe commenting specifically about incremental mode there, assuming it still needs investigation, would help.

Reply:
oops, you're right.  I just wanted to remove `if rotation_count > 0` , but I accidentally removed the following TODO as well. 
```# TODO: Utilize Grover's incremental feature - requires changes to Grover.```